indico/modules/events/timetable/util.py

```diff
@@ -0,0 +1,187 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from collections import defaultdict
+
+from flask import session
+
+from indico.modules.events.timetable.models.entries import TimetableEntry, TimetableEntryType
+from indico.util.date_time import iterdays
+
+
+def serialize_timetable(event, days=None, mgmtMode=False, useAttrCache=False, hideWeekends=False):
```

Comment:
useAttrCache is unused - and you could already snakify the other ones (`hide_weekends`, `management`)?